agent/src/beerocks/slave/ap_manager_thread.cpp

```diff
@@ -34,9 +34,12 @@ using namespace beerocks::net;
 //////////////////////////////////////////////////////////////////////////////
 
 static void copy_radio_supported_channels(std::shared_ptr<bwl::ap_wlan_hal> &ap_wlan_hal,
-                                          beerocks::message::sWifiChannel supported_channels[])
+                                          beerocks::message::sWifiChannel supported_channels[],
+                                          bool use_hw_supported_channels = false)
```

Comment:
like I said before, I don't copy the `hw_supported_channels` into `supported_channels`
since when called I use the notification params' **hw_supported_channels**

Reply:
In that case don't change this method.
just create a dedicated method for copying hw supported channels
`copy_hw_supported_channels()`